.gitignore

```diff
@@ -5,3 +5,4 @@ composer.lock
 .DS_Store
 /sandbox
 docker-compose.yml
+auth.json
```

Comment:
auth.json shouldn't be in .gitignore

Reply:
This is fine. Since we have private repo as a requirement:

```
  "repositories": {
    "repo.magento.com": {
      "type": "composer",
      "url": "https://repo.magento.com/"
    }
  },
```

Running locally ECE-Tools would require credentials.